### ISSUE TITLE
Problem in IE9 - more than 3 images

### ISSUE THREAD

**Issue by apascoalinho** [fredleblanc/roundabout#54]
Hello everyone,

I am using roundabout for the 1st time and I love it.

There is a problem in IE9 I've been tring to find the solution for several days: If I use more than 3 images on the roundabout the webpage "overflows", and there is a lot of "blank page" in the bottom. It only happens in IE9.

The problem is right in the homepage. Please take a look at the diference by using IE9 and then any other browser.

Thanks in advance
Tony

**Comment by fredleblanc** [comment]
I'm looking in IE9 and other browsers and it all looks the same to me. Did you find a fix for this in the last 8 hours or am I unable to reproduce the issue?

**Comment by apascoalinho** [comment]
I found a fix for it by adding <-- ! style="max-height:400px; overflow:hidden --> on the div where the roundabout is.

However, I think you should work on this, because it happens (in IE9) everytime you use more than 3 images.

Basically, the problem is the browser assumes the page height like if the images were displayed vertically on a regular <ul> list, like if the roundabout didn't exist (you can see this before allowing ActiveX to work).

Here's a print screen of how it looked before my tricky fix :) 
http://tinyurl.com/bq8fmg7

**Comment by fredleblanc** [comment]
I still haven't be able to recreate this. I'm not saying that it didn't happen to you, or that things are perfect, I just can't look into things I can't see. :)

**Comment by apascoalinho** [comment]
Have you seen the PDF file? I hope you can understand what was going on.
So can you use like 7 images on the roundabout without having this problem in IE9?

**Comment by fredleblanc** [comment]
I'm looking at the page that you linked to originally. Everything seems to work.

**Comment by apascoalinho** [comment]
Yes, as I told you I was able to fix it, so of course it works perfectly now. 
Please take a look at the print screen file so you can see the issue and understand it.
